examples/60_search/example_parallel_manual_spawning_cli.py

```diff
@@ -175,6 +175,7 @@ def cli_start_worker(scheduler_file_name):
     X_train, X_test, y_train, y_test = \
         sklearn.model_selection.train_test_split(X, y, random_state=1)
     automl = AutoSklearnClassifier(
+        delete_tmp_folder_after_terminate=False,
```

Comment:
Hey, what's the reason for this?

Reply:
This is a very good question. I have made it so, that backend is created on a need basis and destroyed after it is no longer needed. We cannot relay on the __del__ method.

If you look a couple of lines below, we do fit_ensemble. If we do not preserve the backend after fit, the backend directory will be removed.